datafusion/core/tests/sqllogictests/test_files/window.slt

```diff
@@ -3208,8 +3209,8 @@ SELECT
   ORDER BY C3
   LIMIT 5
 ----
-0.970671228336 0.970671228336
```

Comment:
these have the same problem -- that the top 5 values have the same value for the `c3` column